sensor/kubernetes/complianceoperator/handler_impl.go

```diff
@@ -105,6 +105,9 @@ func (m *handlerImpl) ProcessMessage(msg *central.MsgToSensor) error {
 	}
 
 	select {
+	case <-ctx.Done():
+		// TODO(ROX-): Pass this context together with `req` to `m.request`
```

Comment:
Could you open and reference the JIRA ticket here ?

Reply:
Done [5e2ca88](https://github.com/stackrox/stackrox/pull/16064/commits/5e2ca889da88f391bd609738e19a97d3e87b4967)